projects/dsp-ui/src/lib/viewer/representation/still-image/still-image.component.html

```diff
@@ -0,0 +1,57 @@
+<div class="still-image-viewer">
+    <!--<div class="navigation left">
+        <button mat-button class="full-size" id="DSP_OSD_PREV_PAGE">
+            <mat-icon>keyboard_arrow_left</mat-icon>
+        </button>
+    </div>-->
+
+    <!-- main content with navigation and osd viewer -->
+    <div class="content">
+
+        <!-- openseadragon (osd) viewer -->
+        <div class="osd-container"></div>
+        <!-- /openseadragon (osd) -->
+
+        <!-- footer with image caption e.g. copyright information -->
+        <div class="footer">
+            <p class="mat-caption" [innerHtml]="imageCaption"></p>
+        </div>
+
+        <!-- action panel with tools for image -->
+        <mat-toolbar class="action">
+            <mat-toolbar-row>
+                <!-- home button -->
+                <span>
+                    <button mat-icon-button id="DSP_OSD_HOME">
```

Comment:
If this button is only used to restore the image to the original zoom level, maybe we should use a different icon?

Reply:
I also centers the image after dragging it around.

Reply:
I created a DSP task for this https://dasch.myjetbrains.com/youtrack/issue/DSP-518